packages/invoices/src/types.ts

```diff
@@ -157,23 +159,5 @@ export interface InboundShipmentRow extends Omit<InvoiceLine, 'item'> {
 export type InboundShipmentItem = {
   id: string;
   itemId: string;
-  itemCode: string;
-  itemName: string;
-  unitQuantity: number;
-  numberOfPacks: number;
-  locationName?: string | null;
-  itemUnit?: string;
-  batch?: string | null;
-  batches: Record<string, InboundShipmentRow>;
-  sellPrice?: number | undefined;
-  packSize?: number | undefined;
-  note?: string | null;
-  isDeleted?: boolean;
+  lines: [InvoiceLine, ...InvoiceLine[]];
```

Comment:
I'm not even sure this one is needed. Could maybe just make a generic 

```
interface GroupedRow<T> { 
    id: string
    subRows: [T, ...T[]]
}
```

I thought it might be good to keep the `itemId` and `lines` naming.. but I'm not sure - could just use `subRows[0].item` for the item

Reply:
true, if it helps, I prefer `itemId` and `lines` - that's consistent with other usages